docs/PageTitle/README.md

```diff
@@ -0,0 +1,44 @@
+# Page Title
+
+## Installation
+
+Backpack Android is available through [Jitpack](https://jitpack.io/#Skyscanner/backpack-android). Check the main [Readme](https://github.com/skyscanner/backpack-android#installation) for a complete installation guide.
+
+## Usage
+
+The Page Title component can be used in both XML and Kotlin/Java
+
+Example of a Page Title in XML
+
+```xml
+<net.skyscanner.backpack.pagetitle.BpkPageTitle
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  app:pageTitleText="Page Title"
+  app:pageTitleExpandedTextColor="@color/bpkYellow500"/>
```

Comment:
I'm a bit confused why `app:pageTitleExpandedTextColor` is not under themed props and `pageTitleCollapsedTextColor` is. I think we should list `pageTitleCollapsedTextColor` (with the default value) and remove from Theme Props as this is part of the "normal" props for this component.

Another note, we list all attributes here (minus theme attributes) with the **default** value so it's easier for consumers to see what attributes are available.